browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { color, font } from '@brave/leo/tokens/css/variables'
+import { scoped, global } from '../lib/scoped_css'
+
+export const style = scoped.css`
+  .top-controls {
+    position: absolute;
+    inset-block-start: 4px;
+    inset-inline-end: 4px;
+    min-height: 24px;
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    z-index: 2;
+  }
+
+  .settings {
+    --leo-icon-size: 20px;
+
+    opacity: 0.5;
+    color: #fff;
+    filter: drop-shadow(0px 1px 4px rgba(0, 0, 0, 0.60));
+
+    &:hover {
+      opacity: 0.7;
+      cursor: pointer;
+    }
+  }
+
+  .allow-background-pointer-events {
+    /* This element will allow pointer events to target the background. */
+    pointer-events: none;
+
+    /* But children will not (unless they explicitly allow it). */
+    > :not(.allow-background-pointer-events) {
+      pointer-events: auto;
+    }
+
+    /* And not when a popover is open. When a popover is open, pointer events
+       on a background iframe will not "light-dismiss" the popover. */
```

Comment:
nit: I think the `iframe` here isn't really needed

Reply:
Added some text to clarify - the iframe is the only problematic case since it gets in the way of [light-dismiss](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using#auto_state_and_light_dismiss).